partiql-planner/src/testFixtures/resources/inputs/schema_inferencer/pivot.sql

```diff
@@ -0,0 +1,2 @@
+--#[pivot-00]
+PIVOT t.a AT t.c FROM main.T AS t;
```

Comment:
nit: could consider adding a test for if `t.a` is not a string and if `t.c` is an unknown value